commands/qemu-guest-tools/actions_test.go

```diff
@@ -0,0 +1,261 @@
+package qemuguesttools
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"path/filepath"
+	goruntime "runtime"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
```

Comment:
This is unfortunately standard behaviour of [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) which varies slightly from `go fmt`, by introducing the line break and separating standard libraries from other imports. See e.g. [this issue](https://github.com/golang/go/issues/9922).